src/features/room/queue/ui/RoomQueueCard.module.css

```
@@ -0,0 +1,145 @@
+
+.dragHandle:disabled {
+  opacity: 0.35;
+  cursor: default;
+}
+
+.dragHandleIcon {
+  position: relative;
+  width: 4px;
+  height: 4px;
+  border-radius: 1px;
+  background: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)